code/datums/uplink_item.dm

```diff
@@ -324,15 +327,12 @@ var/list/uplink_items = list()
 	desc = "A fully-loaded Donksoft bullpup submachine gun that fires riot grade rounds with a 20-round magazine."
 	item = /obj/item/weapon/gun/projectile/automatic/c20r/toy
 	cost = 8
-	gamemodes = list(/datum/game_mode/nuclear)
```

Comment:
Almost positive thats what surplus variables are for, its used for "weighted" distribution of crates"